gsk4/src/auto/render_node.rs

```diff
@@ -17,10 +18,37 @@ glib::wrapper! {
 }
 
 impl RenderNode {
-    //#[doc(alias = "gsk_render_node_deserialize")]
-    //pub fn deserialize(bytes: &glib::Bytes, error_func: /*Unimplemented*/FnMut(/*Ignored*/ParseLocation, /*Ignored*/ParseLocation, &glib::Error), user_data: /*Unimplemented*/Option<Fundamental: Pointer>) -> Option<RenderNode> {
-    //    unsafe { TODO: call ffi:gsk_render_node_deserialize() }
-    //}
+    #[doc(alias = "gsk_render_node_deserialize")]
+    pub fn deserialize_with_error_func<P: FnMut(&ParseLocation, &ParseLocation, &glib::Error)>(
```

Comment:
And another variant without error func, or otherwise the naming would be strange

Reply:
Ah that's in another file